File: src/main/java/k31/grc/ast/node/func/FParIdentifierT.java
package k31.grc.ast.node.func;

import k31.grc.ast.node.NodeBase;
import k31.grc.ast.visitor.Visitor;

public class FParIdentifierT extends NodeBase {

	public FParIdentifierT(String text) {
		super(text);
	}

	@Override
	public void accept(Visitor v) {

		v.visit(this);
	}
}

File: DotNetGrc/Grc/Ast/Node/Func/FParIdentifierT.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Grc.Ast.Visitor;

namespace Grc.Ast.Node.Func
{
	public class FParIdentifierT : NodeBase
	{
		public FParIdentifierT(string text) : base(text)
		{
		}

		public override void Accept(IVisitor v)
		{
			v.Visit(this);
		}
	}
}
